pkg/k8s/events/types.go

```diff
@@ -30,7 +42,141 @@ const (
 
 // PubSubMessage represents a common messages abstraction to pass through the PubSub interface
 type PubSubMessage struct {
-	Kind   announcements.Kind
+	Kind   Kind
+	Type   EventType
 	OldObj interface{}
 	NewObj interface{}
 }
+
+// Topic returns the PubSub Topic for the given message.
```

Comment:
nit: I don't think we are consistent throughout our codebase or have formally established a precedent, but should we end all declaration comments with periods? https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences  Personally, I have never been consistent one way or the other :) Just thought I'd mention it for discussion

Reply:
good call out! ya i definitely think we should follow go-style. I wonder if there's a linter for that